synapse/util/ldap_auth_provider.py

```diff
@@ -219,9 +219,8 @@ class _LdapConfig(object):
 
         return ldap_config
 
-    def _ldap_simple_bind(self, server, localpart, password):
-        """ Attempt a simple bind with the credentials
-            given by the user against the LDAP server.
+    def _ldap_internal_bind(self, server, bind_dn, password):
```

Comment:
Prefixing with `_` and calling it `internal` looks like a duplication, why not call it `_ldap_bind`?